tools/acrn-crashlog/acrnprobe/README.rst

```diff
@@ -50,6 +50,19 @@ Syntax
   + polling: run a detecting job with fixed time interval.
   + inotify: monitor the change of file or dir.
 
+- trigger :
+  Essentially, trigger represents one section of content. It could be
+  a file's content, a directory's content, or a memory's content which can be
+  obtained. By monitoring it ``acrnprobe`` could detect the certain events which
+  happened in the system.
+
+- crash :
+  A subtype of event. It often corresponds a crash of programs, system or
```

Comment:
change to ``corresponds to a``

add a comma after ``system,``

Reply:
Done.